model/src/order.rs

```diff
@@ -266,6 +266,7 @@ pub struct OrderMetaData {
     pub executed_sell_amount: BigUint,
     #[serde(with = "serde_with::rust::display_fromstr")]
     pub executed_fee_amount: BigUint,
+    pub invalidated: bool,
```

Comment:
Would it make sense to expose the invalidation time as an Option? Might be useful for debugging purposes to see when an order was invalidated.

Reply:
We don't exactly have the time, just the block number and I don't want to make that table more complicated for potential future debugging. I could see that we would later have some internal routes for getting more debug info.